go/vt/schema/parser.go

```diff
@@ -54,7 +54,7 @@ var (
 	createTableRegexp     = regexp.MustCompile(`(?s)(?i)(CREATE\s+TABLE\s+)` + "`" + `([^` + "`" + `]+)` + "`" + `(\s*[(].*$)`)
 	revertStatementRegexp = regexp.MustCompile(`(?i)^revert\s+([\S]*)$`)
 
-	enumValuesRegexp = regexp.MustCompile("(?i)^enum[(](.*)[)]$")
+	enumValuesRegexp = regexp.MustCompile("(?i)^(enum|set)[(](.*)[)]$")
```

Comment:
Does this change affect anything besides online DDL?

Reply:
Double checking.